.gitignore

```diff
@@ -16,6 +16,10 @@ log/
 target/
 
 # Others
+<<<<<<< HEAD
```

Comment:
i think this is, should be removed and below also(seems like unresolved merge conflict)